src/components/Dashboard/Capacity/CapacityItem.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { DonutChart } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import { InlineLoading } from '../../Loading';
+
+const pfGetUtilizationDonutTooltipContents = d =>
+  `<span class="donut-tooltip-pf" style="white-space: nowrap;">${Math.round(d[0].ratio * 1000) / 10} ${
+    d[0].name
+  }</span>`;
+
+const LoadingCapacityItem = ({ idSuffix, title, LoadingComponent }) => (
+  <div className="kubevirt-capacity__item">
```

Comment:
this layout is already present in `CapacityItem`. Can you please put some ifs there and do the loading design there? This way the layout would be duplicated and harder to modify in the future.

Reply:
done